### PR TITLE
WIP: tcyrus-scss

### PR DESCRIPTION
**Pull request by pefoley2**

### CODE REVIEW

File: intranet/apps/preferences/fields.py [hunk 1]
```diff
@@ -10,22 +10,23 @@ class PhoneField(forms.Field):
...
+            return ""
         value = force_text(value).strip()
         return value
 
+    def prepare_value(self, value):
+        if value == "None":
+            return ""
+        return value
+
     def widget_attrs(self, widget):
```
Comment (tjhsst-ion) [inline intranet/apps/preferences/fields.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/tjcsl/ion/file/2379498864/issues/source?bid=3176164&fileBranchId=3217886#l29)

File: intranet/apps/preferences/fields.py [hunk 2]
```diff
@@ -10,22 +10,23 @@ class PhoneField(forms.Field):
...
 
     def to_python(self, value):
         "Returns a Unicode object."
         if value in self.empty_values:
-            return ''
+            return ""
         value = force_text(value).strip()
         return value
 
+    def prepare_value(self, value):
```
Comment (tjhsst-ion) [inline intranet/apps/preferences/fields.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/tjcsl/ion/file/2379498864/issues/source?bid=3176164&fileBranchId=3217886#l24)

**Comment by pefoley2** [comment]
@tcyrus this should probably be merged sooner rather then later, so there isn't too large a delta to review in one shot.

**Comment by tcyrus** [comment]
@pefoley2, I agree

**Comment by pefoley2** [comment]
@jwoglom, you'll have to review as I know next to nothing about CSS.

**Comment by pefoley2** [comment]
@jwoglom ping! :)

**Pull request merged**